src/views/user/ProfileUserView.jsx

```diff
@@ -88,13 +87,13 @@ class MyProfile extends Component {
     }
 
     saveUsername() {
-        const newusername = this.state.NewUsername;
-        console.log(this.checkAvailibility(newusername))
-        if (this.checkAvailibility(newusername)) {
+        const newUserName = this.state.NewUsername;
```

Comment:
Shouldn't this.state.NewUsername also be camelCase?

Reply:
It should